<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaosInterface</class>
 <widget class="QWidget" name="CaosInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="CApplicationInstaller" name="applicationTab">
      <attribute name="title">
       <string>Application</string>
      </attribute>
     </widget>
     <widget class="CKernelInstaller" name="kernelTab">
      <attribute name="title">
       <string>Kernel</string>
      </attribute>
     </widget>
     <widget class="CFileBrowser" name="fileBrowserTab">
      <attribute name="title">
       <string>File Browser</string>
      </attribute>
     </widget>
     <widget class="CaosTerminal" name="terminalTab">
      <attribute name="title">
       <string>Terminal</string>
      </attribute>
     </widget>
     <widget class="CDebug" name="debugTab">
      <attribute name="title">
       <string>Debug</string>
      </attribute>
     </widget>
     <widget class="Monitor" name="monitorTab">
      <attribute name="title">
       <string>Monitor</string>
      </attribute>
     </widget>
     <widget class="Preferences" name="preferencesTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
     </widget>
     <widget class="About" name="aboutTab">
      <attribute name="title">
       <string>Abou</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CFileBrowser</class>
   <extends>QWidget</extends>
   <header>CSdk/CFileBrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CApplicationInstaller</class>
   <extends>QWidget</extends>
   <header>CSdk/CApplicationInstaller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaosTerminal</class>
   <extends>QWidget</extends>
   <header>CaosTerminal.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Monitor</class>
   <extends>QWidget</extends>
   <header>Monitor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Preferences</class>
   <extends>QWidget</extends>
   <header location="global">Preferences.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>About</class>
   <extends>QWidget</extends>
   <header>About.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CKernelInstaller</class>
   <extends>QWidget</extends>
   <header>CSdk/CKernelInstaller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CDebug</class>
   <extends>QWidget</extends>
   <header>CSdk/CDebug.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
